sphinx_doc/howtos/tutorials/docs/navigation2_on_real_turtlebot3.rst

```diff
@@ -0,0 +1,161 @@
+.. _navigation2-on-real-turtlebot3:
+
+Navigation2 on a Real Turtlebot 3
+*********************************
+
+- `Overview`_
+- `Requirements`_
+- `Tutorial Steps`_
+- `Videos`_
+
+Overview
+========
+
+This tutorial shows how to control and navigate Turtlebot 3 using the ROS2 Navigation2 on a real Turtlebot 3 robot.
+Before completing this tutorials, completing the :ref:`navigation2-with-turtlebot3-in-gazebo`. is highly recommended especially if you are new to ROS and Navigation2.
+
+Turtlebot is a low-cost, personal robot kit with open-source software (ROS2).
+Turtblebot robots are widely supported by the ROS community.
+You can find many applications and examples of Turtlebot projects on the Internet.
+You can find more information about Turtlebot3 `here. <http://emanual.robotis.com/docs/en/platform/turtlebot3/overview/>`_
+
+Links to the robot kits and Github repo.
+
+- `Turtlebot Burger <http://www.robotis.us/turtlebot-3-burger-us/>`_
+- `Turtlebot Waffle <http://www.robotis.us/turtlebot-3-waffle-pi/>`_
+- `Turtlebot 3 Github <https://github.com/ROBOTIS-GIT/turtlebot3>`_
+
+This tutorial consists of two parts. In the first part, you learned how to use Turtlebot 3 robots in simulation (Gazebo).
+In this part, you will learn how to control a real Turtlebot Waffle using Navigation2.
+
+``ROS2 Dashing`` and ``Navigation2 Dashing 0.2.4`` are used to create this tutorial.
+You should be able to do this tutorial using other Navigation2 versions as well.
+
+This tutorial may take about 1-2 hours to complete. 
```

Comment:
`1-2 hours`

... it cant actually take that long, can it?

Reply:
Actually, I was being optimistic. It may take longer :) In my opinion testing anything on any real robot means extra time . The simulation tutorial can be 30 mins.

Reply:
haha ok

Reply:
Ok. If we consider that they already completed the previous tutorial, I guess we can say that this one may take about an hour.